x-pack/test/api_integration/apis/management/cross_cluster_replication/constants.js

```diff
@@ -6,3 +6,19 @@
 
 export const API_BASE_PATH = '/api/cross_cluster_replication';
 export const REMOTE_CLUSTERS_API_BASE_PATH = '/api/remote_clusters';
+
+/**
+ * Those settings have been copied from the cross_cluster_replication/common/constants/settings.js file
```

Comment:
is it possible to import this via traversal? I'm worried about these 2 copies potentially getting out of sync :)